OrbitCore/Capture.cpp

```diff
@@ -542,13 +543,14 @@ void Capture::SaveSession(const std::wstring& a_FileName) {
     }
   }
 
-  std::wstring saveFileName = a_FileName;
+  std::string saveFileName = ws2s(a_FileName);
   if (!EndsWith(a_FileName, L".opr")) {
-    saveFileName += L".opr";
+    saveFileName += ".opr";
   }
 
-  SCOPE_TIMER_LOG(Format(L"Saving Orbit session in %s", saveFileName.c_str()));
-  std::ofstream file(ws2s(saveFileName), std::ios::binary);
+  SCOPE_TIMER_LOG(absl::StrFormat("Saving Orbit session in %s",
```

Comment:
LocalScopeTimer used by SCOPE_TIMER_LOG doesn't seem to have a constructor that accepts std::string, only std::wstring.

Reply:
I removed the wstring c-tor and replaced it with std::sting as well - see below.